src/Microsoft.AspNet.SignalR.JS/jquery.signalR.hubs.js

```diff
@@ -7,6 +7,7 @@
 (function ($, window, undefined) {
     "use strict";
 
+    var nextGuid = 0;
```

Comment:
guid is an interesting name for this

Reply:
It's what jQuery uses... I'm not wedded to it at all.